.github/workflows/ci.yml

```diff
@@ -27,6 +27,8 @@ jobs:
             os-image: windows-2022
             os-name: Windows
             artifact-name: windows-artifacts
+            dotnet-versions-to-install: |
+              3.1

```

Comment:
I'd almost've considered splitting out into another PR, but no big deal. Or I can do, if you like the idea but don't want to!

Reply:
Ideally it would be a separate PR, yes. I just wanted to make my build pass.
I'll extract to a separate PR when I have a moment (maybe not today, and maybe not even tomorrow)

Reply:
Done: #1954